docs/section18/Rule_18-2.md

```diff
@@ -0,0 +1,87 @@
+5. get_zone_target_baseline_system()
+6. get_building_total_lab_exhaust_from_zone_exhaust_fan()
+
+
+**Applicability Checks:**
+
+## Rule Logic:  
+- use the function get_baseline_system_types to get a dictionary of baseline system types and systems: `baseline_system_types_dict = get_baseline_system_types(B-RMI)`
+- use the function get_dict_of_zones_and_terminal_units_served_by_hvac_sys to determine which zones are served by each system system: `zones_and_terminal_unit_list_dict = get_dict_of_zones_and_terminal_units_served_by_hvac_sys(B_RMI)`
+- use the function get_zone_target_baseline_system to get the target baseline system types for each zone.  This will be used when checking for laboratory systems: `target_baseline_systems = get_zone_target_baseline_system(P_RMI, B_RMI)`
+- use the function get_building_total_lab_exhaust_from_zone_exhaust_fan to get the totat lab exhaust air flow from zone exhaust fans.  This will be used when checking for laboratory systems: `lab_zone_exhaust = get_building_total_lab_exhaust_from_zone_exhaust_fan(P_RMI)`
+- create a list of laboratory zones meeting G3_1_1d: `laboratory_zones_meeting_G3_1_1d = [x for x in target_baseline_systems if target_baseline_systems[x]["SYSTEM_ORIGIN"] == G3_1_1d]`
+- evaluate the rule for each HVAC system in the B_RMI: `for hvac_system in B_RMI...heating_ventilating_air_conditioning_systems:`
+	- set the result to FAIL: `result = FAIL`
+	- find the hvac_system type by looking through the baseline_system_types_dict for the hvac_system.id: `for hvac_system_type in baseline_system_types_dict:`
+		- find the actual system type of the system (for example, SYS_5 instead of SYS_5B).  Start by looping through the possible system types: `for possible_type in [SYS_1,SYS_2,SYS_3,SYS_4,SYS_5,SYS_6,SYS_7,SYS_8,SYS_9,SYS_10,SYS_11,SYS_12,SYS_13):`
+			- set system_type to possible_type if the result of using the baseline_system_type_compare function is TRUE: `if baseline_system_type_compare(hvac_system_type,possible_type,FALSE): system_type = possible_type`
+		- see if the hvac_system.id is in the list: `if hvac_system.id in? baseline_system_types_dict[hvac_system_type]:`
+  			- Get the list of zones served by the system from zones_and_terminal_unit_list_dict: `zones_served_by_system = zones_and_terminal_unit_list_dict[hvac_system.id]["ZONE_LIST"]`
+     			- check if the system is a single-zone system type: `if any(system_type == sys_type for sys_type in [SYS_1,SYS_2,SYS_3,SYS_4,SYS_9,SYS_10,SYS_11,SYS_12,SYS_13]):`
+     				- there should only be one zone served by the system.  Check the length of zones_served_by_system: `if len(zones_served_by_system) == 1: result = PASS`
+					- otherwise, FAIL: `else: result = FAIL`
+     			- otherwise it is a multi-zone system:`else:`
+					- for a multi-zone system, in a building without lab zones, the system can pass if:
+						1. all zones are on the same floor
+						2. there are no other systems of the same type that serve zones on the same floor
```

Comment:
There is a function get_dict_of_zones_hvac_sys_serving_specific_floor that returns a list of zones on one floor with their associated HVAC systems. Can we call this function for each floor in the building and verify that there is no more than one multizone system per floor is modeled? If this check fails, then the only excuse is that another system is modeled for all labs in the building.

Reply:
I don't think that this function will significantly reduce complexity.  Most of the complexity in this RDS is due to checking for lab zones